public/docs/ts/latest/guide/animations.jade

```diff
@@ -208,72 +203,71 @@ figure.image-display
 +makeExample('animations/ts/app/hero-list-enter-leave-states.component.ts', 'animationdef')(format=".")
 
 :marked
-  ## Animatable Properties and Units
+  ## Animatable properties and units
 
-  Since Angular's animation support builds on top of Web Animations, we can animate any property
+  Since Angular's animation support builds on top of Web Animations, you can animate any property
   that the browser considers *animatable*. This includes positions, sizes, transforms, colors,
-  borders and many others. The W3C maintains
-  [a list of animatable properties](https://www.w3.org/TR/css3-transitions/#animatable-properties).
+  borders, and many others. The W3C maintains
+  [a list of animatable properties](https://www.w3.org/TR/css3-transitions/#animatable-properties)
+  on its [CSS Transitions page](https://www.w3.org/TR/css3-transitions).
 
-  For positional properties that have a numeric value, we can define a unit by providing
+  For positional properties that have a numeric value, define a unit by providing
   the value as a string with the appropriate suffix:
 
   * `'50px'`
   * `'3em'`
   * `'100%'`
 
-  For most dimensional properties we can also just define a number which is then assumed to be
-  in pixels:
+  Define most dimensional properties as a number in pixels:
```

Comment:
Define -> You can define

Also, "as a number in pixels" makes it sounds like you either can't specify px or must specify px. I think the truth (judging from the previous wording) is that you _can_ specify a string with a px/em/% suffix, or you can just specify a number (which is the equivalent of a string with the px suffix).

Not sure how to phrase this best.

Also, the "most dimensional properties" thing makes me wonder what the heck a dimensional property is.